CHANGES.md

```diff
@@ -29,7 +29,9 @@ Format of the entries must be.
 
 * Improve error message in case Docker is not running at start of installation (DCOS-15890)
 
-* * Stop requiring `ssh_user` attribute in `config.yaml` when using parts of deprecated CLI installer (DCOS_OSS-4613)
+* Stop requiring `ssh_user` attribute in `config.yaml` when using parts of deprecated CLI installer (DCOS_OSS-4613)
```

Comment:
Be careful about making unrelated changes in `CHANGES.md`, it can easily lead to merge conflicts if e.g. another PR that's intended for the same train makes a similar change. I'd recommend leaving it for another PR, or at least separating the unrelated changes into their own commits.

Reply:
ah good call. I'll separate into its own commit.